monsteer/qt/steeringWidget/SteeringWidget.h

```diff
@@ -0,0 +1,82 @@
+/* Copyright (c) 2011-2015, EPFL/Blue Brain Project
+ *                          Ahmet Bilgili <ahmet.bilgili@epfl.ch>
+ *
+ * This file is part of Monsteer <https://github.com/BlueBrain/Monsteer>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#ifndef _SteeringWidget_h_
+#define _SteeringWidget_h_
+
+#include <monsteer/types.h>
+
+#ifndef Q_MOC_RUN
+#  include <monsteer/ui_SteeringWidget.h>
```

Comment:
I remember I had a problem with this before. I did not have some much time to look deeper. I just did what you have done with the Animation Controller before but somehow later signals were not working. It was only working when ui header was there.